students/kevin_t/lesson9/mailroom_OO.py

```diff
@@ -0,0 +1,46 @@
+class Donor:
+    def __init__(self, donor_name, donations = []):
```

Comment:
using mutable objects for default arg value can be very dangerous:
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html